rc/lsp.kak

```diff
@@ -1758,11 +1760,11 @@ define-command -hidden lsp-show-message-info -params 1 -docstring %{
     }
 }
 
-define-command -hidden lsp-show-message-log -params 1 -docstring %{
+define-command -hidden lsp-show-message-log -params 2 -docstring %{
```

Comment:
I'm not yet sure if this update is a good idea (because it might break users and complicate the interface for a niche use case).
If it is worth it, the other lsp-show-message-* should be updated in tandem.

Reply:
When I was testing a setup with 3 different servers, this certainly helped with identifying which server was which in the `*debug*` buffer.

Reply:
ok sorry I just searched on github; no one overrides these commands, so let's just update the other `lsp-show-message-*` commands as well